addons/storyquest_bootstrap/new_storyquest_dialog.tscn

```diff
@@ -50,7 +50,7 @@ text = "Title:"
 unique_name_in_owner = true
 layout_mode = 2
 size_flags_horizontal = 3
-placeholder_text = "My Quest"
+placeholder_text = "Sparkling Unicorn Paradise"
```

Comment:
Alternative: "House of the Dragon"

Reply:
I know the unicorn idea is me, but actually I prefer your dragon-based ones. I think it's a clearer example of the kind of story someone might want to tell.